Cargo.toml

```diff
@@ -1,21 +1,25 @@
 [package]
-name = "chronicle-inx"
+name = "chronicle"
 version = "0.1.0"
-authors = [ "IOTA Stiftung" ]
+authors = ["IOTA Stiftung"]
 edition = "2021"
-description = "IOTA permanode implemented as an IOTA Node Extension (INX)"
+description = "IOTA permanode implemented as an IOTA Node Extension (INX)."
 readme = "README.md"
 repository = "https://github.com/iotaledger/inx-chronicle"
 license = "Apache-2.0"
-keywords = [ "iota", "storage", "permanode", "node", "inx" ]
+keywords = ["iota", "storage", "permanode", "node", "inx"]
 homepage = "https://www.iota.org"
 
-[lib]
-name = "chronicle"
-path = "src/lib.rs"
-
-[[bin]]
-name = "inx-chronicle"
-path = "src/main.rs"
+[workspace]
+default-members = [".", "bin/inx-chronicle"]
+members = [
+    ".",
+    "bin/inx-chronicle",
+]
```

Comment:
Can you explain to me why `default-members` and `members` are the same. I am not too familiar with those fields.

Reply:
I was just experimenting. 😄